### PR TITLE
Add bindActions example to readme

### PR DESCRIPTION
**Pull request by leobetosouza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,6 +218,27 @@ export default connect(
 )(Component);
 ```
 
+### Binding actions to a outside object
```
Comment (matheusml) [inline README.md]: I would change this to: 

`Binding actions outside your application scope`, what you think?

File: README.md [hunk 2]
```diff
@@ -217,6 +218,27 @@ export default connect(
...
+
+It's not usual, but sometimes you need to use yor actions outside from you application scope.
```
Comment (matheusml) [inline README.md]: I would change to:

`If you need to bind the actions, to an external listener, outside the application scope, here's a simple way to do it:`

**Comment by leobetosouza** [comment]
@matheusml done :)

**Comment by matheusml** [comment]
:100:

**Pull request merged**